scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -44,21 +59,42 @@ set.seed(12345)
 ```
 
 
-### Operating on lists with `purrr::map()`
 
-We have already pre-processed count data for the four samples we'll be integrating (i.e., filtered cells, normalized counts, and calculated PCA & UMAP).
-These pre-processed SCE objects, stored as RDS files, are available in the `data/rms/processed/` directory. 
-Files are named according to their library IDs:
+## Directories and files
+
+
+We have already prepared count data for the four libraries we'll be integrating (i.e., filtered cells, normalized counts, and calculated PCA & UMAP).
+These SCE objects, stored as RDS files, are available in the `data/rms/processed/` directory and are named according to their `ScPCA` library IDs:
+
+- `SCPCL000479.rds` (Patient A)
+- `SCPCL000480.rds` (Patient A)
+- `SCPCL000481.rds` (Patient B)
+- `SCPCL000482.rds` (Patient B)
+
+To begin, let's set up our directories and files:
+
+```{r directories}
+# Define directory where processed SCE object to be integrated are stored
+input_dir <- file.path("data", "rms", "processed")
+
+# Define directory to save integrated SCE object to
+output_dir <- file.path("data", "rms", "integrated")
```

Comment:
Stephanie's note to self that this needs an if not exists/`dir.create()`!

Reply:
Actually, @allyhawkins we need to chat quickly about this because your input directory is also named `integrated`! 
There can only be one.

Reply:
While it is true can be only one directory, they can have multiple files in them! We just need to make the link the file, not the directory.

Reply:
I moreso meant here whether it's ok that this directory is used for both input and output items. Though maybe that's fine for separate steps of an analysis in this case; one might naturally feed integrated results into diff exp analysis.
But, if we keep the same directory we'll probably want more informative file names themselves between the two integrated files. The dge input is `rms_all_sce.rds`. 

Maybe we aim for this output = `rms_integrated_subset.rds` and the input to DGE = `rms_integrated_all.rds`?

Reply:
Yes, 100% fine to have a directory that is output from one script and input to the next!

> Maybe we aim for this output = `rms_integrated_subset.rds` and the input to DGE = `rms_integrated_all.rds`?

This sounds good to me.